export/index.template.html

```diff
@@ -956,12 +960,7 @@ <h2>Message log</h2>
        */
       document.addEventListener(
         "DOMContentLoaded",
-        async () => {
-          // If styles are saved in local storage, sanitize and apply them.
-          const styleSettings = TKHQ.getSettings();
-          if (styleSettings) {
-            TKHQ.applySettings(styleSettings);
-          }
+        async function () {
```

Comment:
Minor q: any particular reason for removing arrow function? Guessing it's just a stylistic thing?